supporting_content/attributes/discount_handling.md

```diff
@@ -0,0 +1,88 @@
+# Discount Types
+
+Providers offer various discounting schemes for their service offerings. These discounts typically fall into common types such as:
+
+* Commitment-based discounts
+* Tier-based discounts
+* Negotiated discounts
+* Promotional discounts
+* Partnership-based discounts
```

Comment:
```suggestion
* Usage-based discounts
```

https://cloud.google.com/compute/docs/sustained-use-discounts

Reply:
@flowerinthenight is this the same as what you were thinking or separate? Might be good to add the link in the supporting content as well.

Thanks, @cnharris10!

Reply:
Yes, the same. I'll add the link.